assets/js/app/LocusZoom.js

```diff
@@ -31,15 +31,15 @@ LocusZoom.addInstanceToDivById = function(id, datasource, layout, state){
         inst.mapTo(+region[0], +region[1], +region[2]);
     }
     return inst;
-}
+};
     
 // Automatically detect divs by class and populate them with default LocusZoom instances
 LocusZoom.populate = function(selector, datasource, layout, state) {
     if (typeof selector === "undefined"){
-        selector = ".lz-instance";
+        throw ("LocusZoom.populate selector not defined");
     }
     if (typeof layout === "undefined"){
-        layout = JSON.parse(JSON.stringify(LocusZoom.DefaultLayout));
+        layout = {};
```

Comment:
This just looks a bit funny not having the treatment of the `layout` be the same as `state`. I would have thought the merging with the default layout would happen here.